Panels/TabbedPanels.js

```diff
@@ -15,24 +15,29 @@ import Panels from './Panels';
 
 import componentCss from './TabbedPanels.less';
 
-const SpottableLabeledIcon = Spottable(LabeledIcon);
+const SpottableTab = Spottable('div');
 
 const TabBase = kind({
 	name: 'Tab',
 	styles: {
 		css: componentCss,
 		className: 'tab'
 	},
-	render: ({children, labelPosition, onClick, ...rest}) => {
+	render: ({children, icon, labelPosition, onClick, ...rest}) => {
 		return (
 			<Cell
-				icon="star"
 				{...rest}
-				component={SpottableLabeledIcon}
-				labelPosition={labelPosition}
+				component={SpottableTab}
 				onClick={onClick}
 			>
-				{children}
+				<div className={componentCss.label}>
```

Comment:
Could the `className` be applied to the LabeledIcon, or does that interfere with the other styling? Would be nice to not have to reassign this and add a new node, if it's avoidable.